interface-definitions/include/bgp/neighbor-disable-connected-check.xml.i

```diff
@@ -1,7 +1,7 @@
 <!-- include start from bgp/neighbor-disable-connected-check.xml.i -->
 <leafNode name="disable-connected-check">
   <properties>
-    <help>Disable check to see if eBGP peer address is a connected route</help>
+    <help>Allow peerings between eBGP peer using loopback address</help>
```

Comment:
But In the real life (as I remember) it doesn’t related loopback only interface. 
Can confuse users to use loopback only
For example dummy and loopback work different way.
@dmbaturin any pros/cons?

Reply:
Maybe simply use: `loopback/dummy interface address`?

Reply:
we can move `loopback/dummy interface address` , Do you agree ?

Reply:
@fett0 yeah, please change to: `Allow peerings between eBGP peer using loopback/dummy interface address`

Reply:
Can it be a tunnel interface or VIF? :)

Reply:
Perfect ! I'll add a extend explanation in our documentation , just it case .